guide/dev/tips/logging.md

```diff
@@ -11,6 +11,10 @@ with a default configuration which just works out of the box
 and bindings to the other common libraries (``java.util.logging``, ``log4j``, ...)
 if you prefer one of those.
 
+Modify `etc/org.ops4j.pax.logging.cfg`.
```

Comment:
@geomacy to what extend logging is configurable for OSGi distribution through config files in `brooklyn-logback-includes` module?